conjur-oss/README.md

```diff
@@ -294,7 +294,8 @@ The following table lists the configurable parameters of the Conjur OSS chart an
 
 |Parameter|Description|Default|
 |---------|-----------|-------|
-|`account`|Name of the Conjur account to be used by the Kubernetes authenticator|`"default"`|
+|`account.name`|Name of the Conjur account to be used by the Kubernetes authenticator|`"default"`|
+|`account.create`|Controls wether or not a Conjur account is created automatically after installation|`false`|
```

Comment:
I'd lean towards making account auto-create the default behavior... don't know what everyone else thinks?

Reply:
Per a conversation held outside of Github, we've decided to make account creation an opt-in feature for now, to avoid breaking existing installations.